app/services/extension.js

```diff
@@ -66,13 +67,27 @@ export default Service.extend({
     set(this, 'extension', extension);
     this.loadSchemaFromShorthand();
     this.syncFragmentsToSchema();
+    this.persist();
```

Comment:
Why call persist here?

Reply:
simple fragments weren't being generated on the initial render... maybe I should just call `this.makeSimpleFragments();` directly though